cmd/tier.go

```diff
@@ -106,6 +106,7 @@ func (config *TierConfigMgr) ListTiers() []madmin.TierConfig {
 	return tierCfgs
 }
 
+// Edit edits replaces the credentials of the remote tier specified by tierName with creds.
```

Comment:
// Edit replaces the credentials of the remote tier specified by tierName with creds.